runners/core-java/src/main/java/org/apache/beam/runners/core/triggers/TriggerStateMachines.java

```diff
@@ -60,6 +65,14 @@ public static OnceTriggerStateMachine stateMachineForOnceTrigger(OnceTrigger tri
   @VisibleForTesting
   static class StateMachineConverter {
 
+    /**
+     * Annotation indicating that a method is called via dynamic dispatch, not
+     * unused.
+     */
+    @Retention(RetentionPolicy.SOURCE)
+    @Target(ElementType.METHOD)
+    private @interface DynamicDispatch {}
```

Comment:
why this?

If the annotation is purely to have an annotation to add to your IDE's unused list, maybe reuse the existing findbugs annotation?